docs/metrics.md

```diff
@@ -0,0 +1,16 @@
+# Pipeline Controller Metrics
+
+Following pipeline metrics exposed at `controller-service` on port `9090`
+
+Different kind of exporters as Prometheus, Google Stackdriver and so on could be configured using [observability configuration](../config/config-observability.yaml). 
```

Comment:
There is different kind of exporters available : Prometheus, Google Stackdriver, etc. They can be configured using […].

Reply:
:+1: